release_creation/bundle/requirements/v1.2.latest.requirements.txt

```diff
@@ -1,10 +1,10 @@
-dbt-core~=1.2.0 --no-binary dbt-postgres
-dbt-snowflake~=1.2.0
-dbt-bigquery~=1.2.0
-dbt-redshift~=1.2.0
-dbt-postgres~=1.2.0
-dbt-spark[PyHive,ODBC]~=1.2.0
-dbt-databricks==1.2.3
+git+https://github.com/dbt-labs/dbt-core.git@1.2.latest#egg=dbt-core&subdirectory=core --no-binary dbt-postgres
+git+https://github.com/dbt-labs/dbt-core.git@1.2.latest#egg=dbt-postgres&subdirectory=plugins/postgres
+git+https://github.com/dbt-labs/dbt-bigquery.git@1.2.latest#egg=dbt-bigquery
+git+https://github.com/dbt-labs/dbt-snowflake.git@1.2.latest#egg=dbt-snowflake
+git+https://github.com/dbt-labs/dbt-redshift.git@1.2.latest#egg=dbt-redshift
+git+https://github.com/dbt-labs/dbt-spark.git@1.2.latest#egg=dbt-spark[PyHive,ODBC]
+dbt-databricks~=1.2.3, !=1.2.4
```

Comment:
Was this hard-pinned inadvertently? This seems like a separate issue than re-pointing internal adapter repos from PyPI to GH.